### DISCUSSION TITLE
Weird silhouette found in the wild

### DISCUSSION THREAD

**Discussion by ShortyDev** [Stellarium/stellarium#2292] [Q&A] [answered]
I was exploring the sky in Stellarium, when I found a weird silhouette of some sort of seemingly artificial object. Is this just a photo artifact or is this actually present in the sky?

Could be this be the end of a telescope?

Ra/Dec (approx.): 18h43m15.5s +41°11'54.4
Near Vega

![image](https://user-images.githubusercontent.com/68025740/154864862-d77d1160-0cf1-442c-919d-e4d2d829482b.png)

**Comment by gzotti** [comment, marked as answer]
While this is a screenshot from stellarium-web and this project discusses the desktop version, yes, this is obviously a photographic artifact from a reflection of a telescope mirror in the DSS photographic layer.

**Comment by BLACKLOTUS2022** [comment]
I also found the same thing and there is also another one 
![Uploading Screenshot_20221001-193947.png…]()

**Comment by deonaeddy** [comment]
I also found one
